src/pages/experiments/[experimentId]/plots-and-tables/frequency/components/SelectData.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Form,
+  Radio,
+  Select,
+} from 'antd';
+
+const { Option } = Select;
+const SelectData = (props) => {
+  const { onUpdate, config } = props;
+
+  const changeData = (value) => 1;
+
+  const changeClusters = (val) => {
+    onUpdate({ chosenClusters: val });
+  };
+  return (
+    <>
+      <Form.Item>
+        <Radio.Group onChange={(value) => changeData(value)} value={config.currentData}>
+          <Radio value='sample'>Sample</Radio>
+          <Radio value='metadata1'>Metadata 1</Radio>
+          <Radio value='metadata2'>Metadata 2</Radio>
```

Comment:
Leave this for when you are implementing the multiple-sample support. When you get to it, you will not be hardcoding the values for the radiobuttons like this, you will need to read the possible values from the cell sets properties redux store, ignoring only properties for louvain and scratchpad.